src/MediaManagerBundle/Bridge/Chameleon/DataAccess/MediaItemDataAccess.php

```diff
@@ -420,10 +420,10 @@ private function getResultForQueryAndParameters(
                 $params,
                 $paramTypes
             );
-            $numberOfRecords = $stm->rowCount();
+            $numberOfRecords = (int) $stm->rowCount();
 
             $stm = $this->transformStatementForPaging($stm, $mediaManagerListRequest, $query, $params, $paramTypes);
-            $numberOfPages = ceil($numberOfRecords / $mediaManagerListRequest->getPageSize());
+            $numberOfPages = (int) ceil($numberOfRecords / $mediaManagerListRequest->getPageSize());
```

Comment:
This is a bit dumb, but `ceil(...)` is annotated to return a `float`, so it has to be cast explicitly